ifrs17-template/Report/Reports.ipynb

```diff
@@ -376,7 +376,7 @@
             "source": [
                 "# LIC Actuarial",
                 "\n",
-                "\nThe [Actuarial Liability of Incurred Claims](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/Report/ReportScopes#lic-actuarial-actuarial-liability-for-incurred-claims) report shows figures from Fulfilment Cashflow discounted with current yield curve. "
+                "\nThe [Actuarial Liability of Incurred Claims](https://portal.systemorph.cloud/project/ifrs17/env/v1.0.0/Report/ReportScopes#lic-actuarial-actuarial-liability-for-incurred-claims) report shows figures from Fulfilment Cash flow discounted with current yield curve. "
```

Comment:
Cash Flow capitalization?

Reply:
As discussed, no need to capitalize the Flow.